ntrcard.cpp

```diff
@@ -180,7 +183,16 @@ bool sendCommand(const uint64_t cmd, uint16_t response_len, uint8_t *resp, OpFla
 }
 
 bool init() {
-    platform::resetCard();
+    if (platform::CAN_RESET) {
+        platform::resetCard();
+        state.status = Status::RAW;
```

Comment:
we need to check reset result
```
if (!platform::resetCard()) {
  return false;
}
state.status = Status::RAW;
```

Reply:
Done.

Reply:
FWIW there is a need across both `ntrcard_flasher` and `flashcart_core` to add return result checks.

But that's for another PR.